src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (currentlySelectedTab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs, selectedTabId, onTabSelected,
+}) => {
+  return (
+    <div className="App__buttons">
+      {tabs.map((tab) => {
+        return (
+          <button
+            onClick={(event) => {
+              event.preventDefault();
+              onTabSelected(tab);
```

Comment:
do you need event.preventDefault?
add condition - change shouldn't update if the same tab clicked